### ISSUE TITLE
[BottomNavigationView] Add LabelVisibilityMode LABEL_VISIBILITY_UNSELECTED

### ISSUE THREAD

**Issue by charleston10** [material-components/material-components-android#1494]
The component contains some visualization properties, but does not include the alternative mode for visibility `insibility`

Contains
LABEL_VISIBILITY_AUTO, LABEL_VISIBILITY_SELECTED, LABEL_VISIBILITY_LABELED, LABEL_VISIBILITY_UNLABELED

But not contain LABEL_VISIBILITY_UNSELECTED

In this case when clicking on menu the Icon is showing but not show label

**Comment by dsn5ft** [comment]
I don't think this behavior is in the Material spec for Bottom Navigation. Can you explain why you would want to show the labels only on unselected Bottom Navigation items? That seems counterintuitive.
